### DISCUSSION TITLE
Battery power sensor shows usage despite MacBook plugged in

### DISCUSSION THREAD

**Discussion by wkrt** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#584] [General]
Hey guys,
I'm using AlDente Pro 1.14 on MacBook Pro 14" macOS 12.2, plugged in, charge limit reached (sailing 65% - 75%).
Looking at the Stats app (https://github.com/exelban/stats), it shows 0.67 W battery usage.
<img width="292" alt="Screenshot 2022-02-05 at 18 19 41" src="https://user-images.githubusercontent.com/4383815/152652126-ed686720-f5dc-4494-bacc-fbe4be6ee268.png">

This value varies from time to time, but never gets lower than 0.5 W even without any workload.
How should I interpret this value?
Is my battery continuously charged/discharged even if the charge limit is reached?

**Comment by kentzh1** [comment]
This also confuses me, the 0.08A battery shows that maybe the battery has been working and not using the power fully?

**Comment by MatthiasKerbl** [comment]
I am not sure what this means either, it might just be within the margin of error. I will let you know when I find out more about that.
